File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F13.P2.go
package main

import (
	"fmt"
	"strconv"
)

func recovered(s string) int32 {
	// BEGIN PROBE F13.P2
	v, err := strconv.ParseInt(s, 10, 32)
	var n int32
	if err == nil {
		n = int32(v)
	}
	return n
	// END PROBE F13.P2
}

func main() {
	fmt.Println(recovered("21"))
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F13.P2.ts
function parseOrZero(s: string): number {
// BEGIN PROBE F13.P2
const p = parseInt(s, 10)
const n = Number.isNaN(p) ? 0 : p
return n
// END PROBE F13.P2
}

console.log(parseOrZero("21"))
